### PR TITLE
Calibration vignette

### PR DESCRIPTION
**Pull request by InigoP**
Update to calibration vignette and corresponding pkgdown website

### CODE REVIEW

File: vignettes/calibration-2021.Rmd [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+keep <- c(1, 3)
+knitr::kable(winterRunDSM::watershed_attributes[keep, "watershed", drop = FALSE])
+```
+
+
+## Packages
+
+```{r, message=FALSE}
+library(GA)
+library(winterRunDSM)
```
Comment (ergz) [inline vignettes/calibration-2021.Rmd:51]: this is generating a warning please suppress warning in the r code chunk header

File: vignettes/calibration-2021.Rmd [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+### GrandTab
+
+We will be minimizing the error between [GrandTab](https://wildlife.ca.gov/Conservation/Fishes/Chinook-Salmon/Anadromous-Assessment) estimated escapement data for the years 1998-2017 to natural spawners predicted by the DSM. We are further scaling this error by a weight representing the amount of observed data available for a given watershed. Given that
+GrandTab includes both hatchery and natural spawners, we will scale GrandTab escapement
+values using an estimated proportion natural spawners for each watershed.
+This will allow us to compare predicted natural spawners to psuedo GrandTab natural spawners.
+
+For calibration, we have prepared the GrandTab data in two ways:
+
+1. `DSMCalibrationData::grandtab_observed` - This data will be used in the fitness function to measure the difference between model predictions and observed escapement. Missing values are NA and we made all records less than 100 NA to account for a lack of confidence for counts less than 100.
```
Comment (ergz) [inline vignettes/calibration-2021.Rmd:71]: when the vignette is rendered these are displayed as clickable links but they lead to no site, can you check if there is  a way we can link to the docs page of `DSMCalibrationData::grandtab_observed` ?

File: vignettes/calibration-2021.Rmd [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+                        winterRunDSM::pretty_num(spawners))) %>% 
+  config(displayModeBar = FALSE)
+```
+
+### Selected Proxy Years
+
+The data inputs to the DSM are for years 1980-1999. We selected proxy years for 1998-2017
+from the 1980-1999 model inputs by [comparing the DWR water year indices](https://cdec.water.ca.gov/reportapp/javareports?name=WSIHIST) in this [analysis](https://github.com/CVPIA-OSC/DSMCalibrationData/blob/main/data-raw/synthetic_years.R).
+
+These inputs are generated by calling `DSMCalibrationData::set_synth_years(params)`, with a `params` object from the life cycle model ex: `fallRunDSM::params`. See the documentation for more details on the [DSMCalibrationData](https://github.com/CVPIA-OSC/DSMCalibrationData)
```
Comment (ergz) [inline vignettes/calibration-2021.Rmd]: change the reference to fallRunDSM:: to winterRunDSM::

**Review by ergz — COMMENTED** [review]
please review the comments

File: vignettes/calibration-2021.Rmd (hunk 1 above)
Comment (InigoP) [reply vignettes/calibration-2021.Rmd:51]: fixed

File: vignettes/calibration-2021.Rmd (hunk 2 above)
Comment (InigoP) [reply vignettes/calibration-2021.Rmd:71]: No docs page for DSMCalibrationData::grandtab_observed; will push it for another time

File: vignettes/calibration-2021.Rmd (hunk 3 above)
Comment (InigoP) [reply vignettes/calibration-2021.Rmd]: fixed

**Pull request merged**